pkg/internal/config/config.go

```diff
@@ -55,6 +56,7 @@ const (
 	TrafficGeneratorImageDefault                      = "quay.io/kiagnose/kubevirt-dpdk-checkup-traffic-gen:latest"
 	VMContainerDiskImageDefault                       = "quay.io/kiagnose/kubevirt-dpdk-checkup-vm:latest"
 	TestDurationDefault                               = 5 * time.Minute
+	VerboseDefault                                    = false
```

Comment:
This is redundant IMO.

Reply:
you're right but I want it to explicitly state that's what I expect to get